### PR TITLE
add coverage

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@scopatz I guess coverage/codecov doesn't play nice with `.xsh` files? Or is there some flag I can set to cover those files?

File: run_tests.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+if __name__ == '__main__':
+    # show output results from every test function
+    args = ['-v', 'tests']
+    # show the message output for skipped and expected failure tests
+    if len(sys.argv) > 1:
+        args.extend(sys.argv[1:])
+    print('pytest arguments: {}'.format(args))
+    # # compute coverage stats for xpdAcq
+    # args.extend(['--cov', 'xpdAcq'])
```
Comment (justcalamari) [inline run_tests.py]: Can you remove references to xpdAcq?

**Comment by scopatz** [comment]
@CJ-Wright - To the best of my knowledge, no.

**Comment by scopatz** [comment]
But I have never really used coverage, so I am not super familiar

**Comment by CJ-Wright** [comment]
It seems like there is a coverage plugin system so maybe we need a xonsh plugin.

**Comment by scopatz** [comment]
Yeah that would be great!

**Pull request merged**